### PR TITLE
hotfix/room-overview

### PR DESCRIPTION
**Pull request by UzaeirKhan**
# Description
Hotfix for Martin's PR. Tests were failing because selector were hidden in the `div`

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: page-objects/pages/TASKPage.js [hunk 1]
```diff
@@ -16,12 +16,13 @@ const studentSubmissionTab = "//a[@id='submission-tab-link' and contains(.,  'Ab
 const remoteFilePathInput = 'input[type=file][class=dz-hidden-input]';
 const commentBtn = '#comment-tab-link';
 const commentGradingTabSel = '#feedback-tab-link';
-const hometasksTabSel = 'button[data-testid="hometasks"]';
+const createTaskAndTopicBtn = "//button//span//div//span[contains(., 'Erstellen')]";
```
Comment (Loki-Afro) [inline page-objects/pages/TASKPage.js:19]: isn't that language specific now?

File: step_definitions/tasks/createTask_steps.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ When(/^.* clicks Create-a-task button in the course '([^'].*)'$/, async function
 	await navigationLeftPage.clickNavItemRoomsOverview();
 	await roomsOverview.clickOnTheElementWithName(coursename);
 	await TASKPage.gotoTasksTab();
-	await TASKListPage.clickCreateTaskButtonInTheCourse();
+	//await TASKListPage.clickCreateTaskButtonInTheCourse();
```
Comment (Loki-Afro) [inline step_definitions/tasks/createTask_steps.js:22]: remove dead code.

**Review by Loki-Afro — CHANGES_REQUESTED** [review]

File: step_definitions/tasks/createTask_steps.js (hunk 2 above)
Comment (UzaeirKhan) [reply step_definitions/tasks/createTask_steps.js]: Agreed... removed

File: page-objects/pages/TASKPage.js (hunk 1 above)
Comment (UzaeirKhan) [reply page-objects/pages/TASKPage.js]: No, its not language specific.

**Pull request closed without merging**